code/modules/mining/minebot.dm

```diff
@@ -61,6 +61,11 @@
 
 	SetCollectBehavior()
 
+/mob/living/simple_animal/hostile/mining_drone/emp_act(severity)
+	adjustFireLoss(100 / severity)
```

Comment:
🤔 Well the mining bot took fire damage when I tested it...

Regardless, I have changed what you asked.